app/views/volunteers/_volunteer.html.slim

```diff
@@ -15,8 +15,11 @@ tr
   td = volunteer.interests
   td = t(".acceptance.#{volunteer.acceptance}")
   td = volunteer.external? ? t_attr(:external) : t_attr(:internal)
-  td = volunteer.self_applicant? ? t('volunteer_self_applicant') : volunteer.registrar.full_name
-  - if action_name == 'index'
+  - if volunteer.self_applicant?
+    td= t('volunteer_self_applicant')
```

Comment:
```slim
td
  - if volunteer.self_applicant?
    = t('volunteer_self_applicant')
  - else
    = link_to volunteer.registrar.full_name, url_for(volunteer.registrar.profile || volunteer.registrar)
```
If different content is going to the same element, nest conditionals, not html element -> avoid indent confusion...